pkg/scheduler/preemption/preemption_test.go

```diff
@@ -2067,7 +2067,7 @@ func TestFairPreemptions(t *testing.T) {
 			}
 			wlInfo := workload.NewInfo(tc.incoming)
 			wlInfo.ClusterQueue = tc.targetCQ
-			targets := preemptor.GetTargets(log, *wlInfo, singlePodSetAssignment(
+			_, targets := preemptor.GetTargets(log, *wlInfo, singlePodSetAssignment(
```

Comment:
Regarding newAssignment verification.

Reply:
refactored in https://github.com/kubernetes-sigs/kueue/pull/4418/commits/f1a3f568c6a062fc5c4491e918957574fb032fa6 which keeps the interface of GetTargets the same